src/tests/specs/quickPreview.spec.ts

```diff
@@ -32,6 +32,12 @@ describe('planner quick preview tests', () => {
     expect(await planner.quickPreview.hasLabel(c.newLabel)).toBeTruthy();
   });
 
+  it('should open quick preview and edit the title',async () => {
+    await planner.workItemList.clickWorkItem(c.workItemTitle1);
+    await planner.quickPreview.updateTitle(c.editWorkItemTitle1);
+    expect(await this.quickPreview.titleDiv.getTextWhenReady()).toBe('Title Text "<0>"');
```

Comment:
@sahil143 change this to planner.quickPreview.titleDiv.getTextWhenReady()).toBe('Title Text "<0>"');